src/components/sectionList/translation/translation.module.css

```diff
@@ -13,27 +20,21 @@
     justify-content: space-between;
     background-color: var(--colors-grey200);
     color: var(--colors-grey600);
-    padding: 5px 10px;
+    padding-block: 5px;
+    padding-inline: 10px;
 }
-.formObj {
+.translateableFields {
     display: flex;
     align-items: center;
-    margin-right: 10px;
```

Comment:
did you want to get rid of this or replace it with `margin-inline-end`? Just wondering because you mentioned in your PR comment updating `margin-right` with equivalent logical-property css.

Reply:
Hehe, thanks for noticing! I actually did notice this, but apparantly didn't push my branch...